applications/portal/backend/api/helpers/generate_population_cells.py

```diff
@@ -1,20 +1,39 @@
 import os.path
 import uuid
 
-
 from api.services.filesystem_service import move_file
 
 OUTPUT_DIRECTORY = "./out"
-CELLS_CSV = "cells_with_labels"
+PAIR_CELLS_CSV_SUFFIXE = "cells_with_labels"
```

Comment:
@afonsobspinto why is there a diff between a single and a pair file? Can't we come up with something that is auto detecting the "file" type  (single/pair)